### ISSUE TITLE
Add setup option which allows to import additional templates after base install

### ISSUE THREAD

**Issue by yurem** [GluuFederation/community-edition-setup#285]
We need to allow run:
```
./setup.py -import-ldif custom-ldif-directory
```

This command should render all ldif templates in specified directory and loads them to LDAP

**Comment by ganesh-at-wiw** [comment]
# Note: This procedure MUST be run only AFTER Base Install.

Added the code to setup.py as per below github issue history.

https://github.com/GluuFederation/community-edition-setup/commit/a0eb74ad62b1a0e85b0c5257ad9d79b172405dfe#diff-2eeaed663bd0d25b7e608891384b7298


Here is brief way to run the import-ldif:
1. Since base install is already done, so we need setup.properties. Hence do this to copy setup.properties, inside **/install/community-edition-setup** in **chroot** :
`cp setup.properties.last setup.properties`
2.  Inside **/install/community-edition-setup** in **chroot** place your templates inside your custom directory say: my_custom_templates. 
```
GLUU.root@vsurpr0001:/install/community-edition-setup# ls my_custom_templates
scim-test-data.ldif
GLUU.root@vsurpr0001:/install/community-edition-setup# 
```
3. Run the import-ldif option as:
`./setup.py --import-ldif=my_custom_templates`

The output directory for rendered ldif files from rendered template will be having extension .output  after the name of directory mentioned. my_custom_templates.output in current case.
See below output:
```
GLUU.root@vsurpr0001:/install/community-edition-setup# ls my_custom_templates.output/
scim-test-data.ldif
GLUU.root@vsurpr0001:/install/community-edition-setup# 
```

The output of setup.log will be like:

> 19:59:47 07/25/17 Removed ./setup.log
> 19:59:47 07/25/17 Removed ./setup_error.log
> 19:59:47 07/25/17 Installing Gluu Server
> 19:59:47 07/25/17 ./setup.properties Properties found!
> 
> 19:59:47 07/25/17 Loading Properties ./setup.properties
> 19:59:47 07/25/17 Checking properties
> 19:59:47 07/25/17 Rendering custom templates
> 19:59:47 07/25/17 Rendering custom templates from my_custom_templates to my_custom_templates.output
> 19:59:47 07/25/17 Running: /bin/mkdir -p my_custom_templates.output
> 19:59:47 07/25/17 Rendering template scim-test-data.ldif
> 19:59:47 07/25/17 Importing Custom LDIF files into OpenLDAP
> 19:59:47 07/25/17 Running: /bin/su ldap -c cd /install/community-edition-setup; /opt/symas/bin/slapadd -b o=gluu -f /opt/symas/etc/openldap/slapd.conf -l my_custom_templates.output/scim-test-data.ldif
>
